src/main/java/Duke.java

```diff
@@ -1,10 +1,216 @@
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by FernFlower decompiler)
+//
+
+import exception.commandException;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.Scanner;
+import task.Deadlines;
+import task.Events;
+import task.Task;
+import task.ToDos;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private static ArrayList<Task> listOfTasks = new ArrayList();
+
+    public Duke() {
+    }
+
+    public static void main(String[] args) throws commandException, IOException, ClassNotFoundException {
+        Scanner in = new Scanner(System.in);
+        System.out.println("Hello! I am Groot");
+        System.out.println("What can I do for you?");
+        System.out.println("Please find your previous list items below: ");
+        int taskIndex=read("/Users/nikki/Documents/GitHub/ip/src/main/java/output.txt");
+        String userInput = in.nextLine();
+
+        while(true) {
+            try {
+                if (userInput.equalsIgnoreCase("bye")) {
+                    System.out.println("Bye. Hope to see you again soon!");
+                    break;
+                }
+
+                int slashIndex;
+                if (userInput.equalsIgnoreCase("list")) {
+                    if (!userInput.equals("list")) {
+                        throw new commandException();
+                    }
+
+                    System.out.println("Here are the tasks in your list: ");
+
+                    for(int index = 0; index < listOfTasks.size(); ++index) {
+                        int j = index + 1;
+                        System.out.println(j + "." + ((Task)listOfTasks.get(index)).getType() +
+                                ((Task)listOfTasks.get(index)).getMark() + " " +
+                                ((Task)listOfTasks.get(index)).toString());
+                    }
+                } else {
+                    PrintStream var10000;
```

Comment:
Maybe you should avoid naming the variable with a meaningless name like "var10000".